src/tests/dune

```diff
@@ -25,6 +25,7 @@
     osemgrep_targeting
     osemgrep_language_server
   )
+ (modes native byte)
```

Comment:
maybe add a comment about byte, which is it allows to produce a test.bc and use
earlybird on it